### PR TITLE
[WIP] use grpc standard health check in python client

### LINKED ISSUE #22: use new standard health check in python client
After implementing the standard grpc health check (#19) we should use it everywhere.

### PR DESCRIPTION
**Pull request by fm3**
@jstriebel I’m not really satisfied with the lambda solution anymore, since the method now uses the two stubs. Also, 
```
    if not reply.success:
        sys.exit(1)
```
doesn’t work anymore since HealthCheckReply does not have a success field.
Do you have any ideas / opinions?

fixes #22

### CODE REVIEW

**Comment by jstriebel** [comment]
@fm3
For the lambdas: feel free to use regular functions.
For the health check: can't we just check that `ServingStatus` is 1? See https://github.com/grpc/grpc/blob/master/src/proto/grpc/health/v1/health.proto

**Comment by fm3** [comment]
Sure, I was just hesitant to create another case distinction. But I’ll just go ahead then.

File: client/fossildb-client [hunk 1]
```diff
@@ -43,12 +44,27 @@ def main():
 
     channel = grpc.insecure_channel(full_address)
     stub = proto_rpc.FossilDBStub(channel)
+    healthStub = health_pb2_grpc.HealthStub(channel)
```
Comment (jstriebel) [inline client/fossildb-client]: Please move the stubs to their respective if-case.

File: client/fossildb-client [hunk 2]
```diff
@@ -15,8 +17,7 @@ def main():
             stub.Backup(proto.BackupRequest()),
         'restore': lambda stub:
             stub.RestoreFromBackup(proto.RestoreFromBackupRequest()),
-        'health': lambda stub:
-            stub.Health(proto.HealthRequest())
+        'health': None
```
Comment (jstriebel) [inline client/fossildb-client]: This doesn't make sense anymore. Maybe remove the stub argument and replace (for our stub) with a function used in backup & restore. Also health could still be part of that map. I'll prepare a commit with a proposal.

**Comment by jstriebel** [comment]
@fm3: See #26

**Pull request merged**